src/patternfly/components/DataList/data-list.scss

```
@@ -307,7 +303,7 @@
   transition: var(--pf-c-data-list__toggle-icon--Transition);
 
   .pf-c-data-list__item.pf-m-expanded & {
-    transform: var(--pf-c-data-list__item--m-expanded__toggle--c-button-icon--Transform);
+    transform: var(--pf-c-data-list__item--m-expanded__toggle--c-button__toggle-icon--Transform);
```

shouldn't this just be

```suggestion
    transform: var(--pf-c-data-list__item--m-expanded__toggle-icon--Transform);
```